src/main/java/com/google/devtools/build/lib/analysis/starlark/StarlarkAttributesCollection.java

```diff
@@ -230,6 +231,13 @@ public static Object convertAttributeValue(
         return null;
       }
 
+      // Don't expose exec_group_compatible_with to Starlark. There is no reason for it to be used
+      // by the rule implementation function and it's type (LABEL_LIST_DICT) is not available to
```

Comment:
Nit: "it's" -> "its". If updating the middle of the commit stack is tricky I'll handle on import.

Reply:
Fixed, `git-revise` is great at this.